perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -272,9 +272,11 @@ sub by_category_ajax_data : Private {
     }
 
     my $non_public = $c->stash->{non_public_categories}->{$category};
+    my $anon_button = ($c->cobrand->allow_anonymous_reports($category) eq 'button');
```

Comment:
I wonder if there's a way to pass a contact here instead of just the category name - it'd allow for more expressive things down the line e.g. anonymous reports on .com for a specific body, storing the anonymous-allowed flag in contact `extra` (and admin UI to control it), only allowing anonymous reports if the send_method is email, all that sort of thing.

Reply:
See what you mean, but annoyingly contact is elusive, given technically (though unlikely/never going to happen) it could be an overlap category with two contacts from two different bodies.

Reply:
Ah well, nevermind :)